### PR TITLE
Update windows-store-for-business.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the important note about the retirement of the Microsoft store for business

Fixes https://github.com/MicrosoftDocs/memdocs/issues/3815

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/apps/windows-store-for-business.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ The [Microsoft Store for Business](https://www.microsoft.com/business-store) giv
 ## Before you start
 
 > [!IMPORTANT]
-> Microsoft Store for Business will be retired in the first quarter of 2023. However, admins can still leverage the connection to Store for Business and Education from their UEM solution to deploy apps to managed Windows 11 devices until they are retired in 2023.
+> The retirement of the Microsoft Store for Business and the Microsoft Store for Education, originally scheduled for March 31, 2023, has been postponed. Once retired,  admins can still leverage the connection to Store for Business and Education from their UEM solution to deploy apps to managed Windows 11 devices until they are retired in 2023.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/windows-store-for-business.md:50]:
```suggestion
> The retirement of the Microsoft Store for Business and the Microsoft Store for Education, originally scheduled for March 31, 2023, has been postponed. Until they are retired, admins can still leverage the connection to Store for Business and Education from their UEM solution to deploy apps to managed Windows 11 devices.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
Thank you! #sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**